dev-docs/docs/introduction/contributing.mdx

```diff
@@ -26,11 +26,11 @@ In case you want to pick up something from the roadmap, comment on that issue an
 ### Option 2 - CodeSandbox
 
 1. Go to https://codesandbox.io/p/github/excalidraw/excalidraw
-1. Connect your GitHub account
-1. Go to Git tab on left side
-1. Tap on `Fork Sandbox`
-1. Write your code
-1. Commit and PR automatically
+1. Connect your GitHub account (Sign in on the top left corner)
```

Comment:
```suggestion
1. Connect your GitHub account
```
I think this is intutive as users would be aware of how to connect their GH account